contracts/liquid-locking/src/lib.rs

```diff
@@ -34,7 +34,12 @@ pub trait LiquidLocking {
     #[payable("*")]
     #[endpoint]
     fn lock(&self) {
+        require!(
```

Comment:
Are there users that also sent egld to the contract, besides the ESDTs?

Reply:
nope.. so ESDT should be set there